src/main/java/Storage.java

```diff
@@ -0,0 +1,70 @@
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Scanner;
+
+/**
+ * Represents a storage of the task list.
+ */
+public class Storage {
+
+    protected String filePath;
+
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+    /**
+     * Retrieve the list of tasks from local file.
+     * @return A list of current tasks in record.
+     */
+    public List<Task> load() {
+        Path file = Paths.get(filePath);
+        List<Task> tasks = new LinkedList<>();
```

Comment:
I'm curious why you ended up using a LinkedList. Potentially, an ArrayList may be more efficient, especially when randomly accessing various parts of the array, and often not in sequence? Then again, modification of list may be more efficient using a LinkedList.